client/components/main/layouts.jade

```diff
@@ -13,6 +13,13 @@ template(name="userFormsLayout")
     h1.at-form-landing-logo
       img(src="{{pathFor '/wekan-logo.png'}}" alt="Wekan")
     +Template.dynamic(template=content)
+    div.at-form-lang
+      select.select-lang.js-userform-set-language
```

Comment:
I now understand what you mean. 

A welcome board might be helpful to new user, maybe a simple start board with 4 lists (PLAN / TODO / DOING / DONE) is already enough. If too heavy or too many i18n sample data, maybe it will become a burden.